packages/playwright-ct-core/types/component.d.ts

```diff
@@ -14,11 +14,6 @@
  * limitations under the License.
  */
 
-type JsonPrimitive = string | number | boolean | null;
-type JsonValue = JsonPrimitive | JsonObject | JsonArray;
-type JsonArray = JsonValue[];
-export type JsonObject = { [Key in string]?: JsonValue };
```

Comment:
NOTE: `JsonObject` becomes obsolete because components can be passed to `hooksConfig`.